index.html

```diff
@@ -14,6 +15,129 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+
+  <div class="container">
+
+    <nav>
+
+      <a href="#">Home</a>
+      <a href="#">About</a>
+      <a href="portfolio.html">Portfolio</a>
+      <a href="#">Contact</a>
+
+    </nav>
+
+    <section class="top">
+
+      <img src="img/Dr_Evil.jpg" alt="Dr Evil">
+
+      <h1>Dr Evil - Evil Scientist</h1>
+
+      <p>World domination is my jam, baby!</p>
+
+    </section>
+
+    <section class="team-up">
+
+      <p>Ocassionally I have time to work on side projects, if you're interested, let me know by signing up on this website.</p><br>
+
+      <button onclick="location.href='mailto:drevil@whitehouse.gov';">Let's team up!</button>
```

Comment:
Make [most] buttons have a pointer on hover! In your CSS, use `cursor: pointer`. Non-pointer-on-hover buttons make me depressed (unless they're a specific part of a design).